<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_openDB"/>
   <addaction name="action_add"/>
   <addaction name="action_insert"/>
   <addaction name="action_del"/>
   <addaction name="separator"/>
   <addaction name="action_save"/>
   <addaction name="action_cancel"/>
   <addaction name="separator"/>
   <addaction name="action_listRecord"/>
   <addaction name="action_quit"/>
  </widget>
  <action name="action_openDB">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开数据库文件</string>
   </property>
  </action>
  <action name="action_add">
   <property name="text">
    <string>添加</string>
   </property>
   <property name="toolTip">
    <string>添加记录</string>
   </property>
  </action>
  <action name="action_insert">
   <property name="text">
    <string>插入</string>
   </property>
   <property name="toolTip">
    <string>插入记录</string>
   </property>
  </action>
  <action name="action_del">
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除记录</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存文件</string>
   </property>
  </action>
  <action name="action_cancel">
   <property name="text">
    <string>取消</string>
   </property>
   <property name="toolTip">
    <string>取消更改</string>
   </property>
  </action>
  <action name="action_listRecord">
   <property name="text">
    <string>字段列表</string>
   </property>
   <property name="toolTip">
    <string>列出各个字段</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出系统</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
